packages/base-types/src/models/project.ts

```diff
@@ -27,6 +27,8 @@ export interface ProjectPrototype {
   trainedModel?: PrototypeModel;
   lastTrainedTime?: number;
   nlp?: ProjectPrototypeNLP;
+  delayEnabled: boolean;
+  delay: number;
```

Comment:
please include a duration in this, ex.
```suggestion
  delaySeconds: number;
```